### ISSUE TITLE
enable_cluster_plan打开时，create table as select方式从一个复制表创建新表时，报错

### ISSUE THREAD

**Issue by mass3** [ADBSQL/AntDB#32]
### enable_cluster_plan打开时，create table as select方式从一个复制表创建新表时，报错
postgres=# create table atacc1 (a int4 not null, b int4, c int4 not null, d int4) with oids distribute by replication;
CREATE TABLE
postgres=# insert into atacc1 values (1, 2, 3, 4);
INSERT 0 1
postgres=# insert into atacc1 values (21, 22, 23);
INSERT 0 1
**postgres=# create table test1 as select * from atacc1;
ERROR:  not support replicate to distribute yet!**
postgres=# 
postgres=# set enable_cluster_plan = off;
SET
postgres=# create table test1 as select * from atacc1;
INSERT 0 2
postgres=# 

### 测试sql：
```
create table atacc1 (a int4 not null, b int4, c int4 not null, d int4) with oids distribute by replication;
insert into atacc1 values (1, 2, 3, 4);
insert into atacc1 values (21, 22, 23);
create table test1 as select * from atacc1;
```

**Comment by hsiaoby** [comment]
fixed by acb13f511adb0fffaa74608d6de7ddd788ad6303

**Comment by mass3** [comment]
从复制表创建新表成功
```
postgres=# create table atacc1 (a int4 not null, b int4, c int4 not null, d int4) with oids distribute by replication;
insert into atacc1 values (1, 2, 3, 4);
insert into atacc1 values (21, 22, 23);
create table test1 as select * from atacc1;
CREATE TABLE
postgres=# insert into atacc1 values (1, 2, 3, 4);
INSERT 0 1
postgres=# insert into atacc1 values (21, 22, 23);
INSERT 0 1
postgres=# create table test1 as select * from atacc1;
INSERT 0 2
postgres=# 
```
